README.md

```diff
@@ -67,10 +75,28 @@ Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawV
 
 # Usage
 
+* [Overriding configuration on runtime](#runtime-conf)
 * [Load the Mini App list](#load-miniapp-list)
 * [Get a MiniAppInfo](#get-mini-appinfo)
 * [Create a MiniApp](#create-mini-app)
 
+<div id="runtime-conf"></div>
+
+### Overriding configuration on runtime
+
+Every call to the API can be done with default parameters retrieved from the project .plist configuratin file, or by providing a `MiniAppSdkConfig` object during the call. Here is a simple example class we use to create the configuration in samples below:
```

Comment:
configuration?

Reply:
if you take a look at simple code in the documentation we are providing an example for custom configuration calls. Did I make a mistake in the description ? Please tell me an other way to write this

Reply:
'configuration' is misspelled in the comments